Add KubePersistentVolumeFillingUp runbook

pracucci opened the pull request:
#### What this PR does

`KubePersistentVolumeFillingUp` is not a Mimir alert, but it's a very common one when deploying Mimir in K8S. In this PR I propose to upstream a slightly modified version of an internal runbook we have at Grafana Labs for this alert.

Note: I've added @aknuds1 as `Co-authored-by` in the commit, because it did most of the work in the internal runbook and I don't want such credit to be lost.

#### Which issue(s) this PR fixes or relates to

N/A

#### Checklist

- [ ] Tests updated.
- [x] Documentation added.
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

jhesketh commented:
> Note: I've added @aknuds1 as `Signed-off-by` in the commit, because it did most of the work in the internal runbook and I don't want such credit to be lost.

(In case you weren't aware, you can use [Co-authored-by](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors#creating-co-authored-commits-on-the-command-line) in commit messages to credit multiple authors. (But maybe you meant signed off specifically))

jhesketh reviewed (approved):
lgtm

pracucci commented:
> > Note: I've added @aknuds1 as `Signed-off-by` in the commit, because it did most of the work in the internal runbook and I don't want such credit to be lost.
> 
> (In case you weren't aware, you can use [Co-authored-by](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors#creating-co-authored-commits-on-the-command-line) in commit messages to credit multiple authors. (But maybe you meant signed off specifically))

Actually it was my mistake. I wanted to use `Co-authored-by`. Thanks for pointing out. I've fixed it.
